autogpt_platform/backend/backend/data/graph.py

```diff
@@ -88,8 +89,7 @@ def from_db(node: AgentNode, for_export: bool = False) -> "NodeModel":
         return obj
 
     def is_triggered_by_event_type(self, event_type: str) -> bool:
-        if not (block := get_block(self.block_id)):
-            raise ValueError(f"Block #{self.block_id} not found for node #{self.id}")
+        block = self.block
```

Comment:
this is unhinged syntax

Reply:
please be more specific

Reply:
block = self.block running code in the background subverts my expectations heavily that this could raise

Reply:
So it should be a method instead of a @property?